### DISCUSSION TITLE
Any way to style the page changer components? Adding customizing font size option for docs.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#934] [General]
This 
![Screenshot 2024-10-06 133919](https://github.com/user-attachments/assets/3b74c372-9d1a-44f5-bea3-2bc195bd40b5)

--- 
I have created a component which changes the font size of the components in mdx and works great. You can check it at https://mindect.vercel.app/learn/sl. Should I contribute that in fumadocs.

**Comment by fuma-nama** [comment]
> Adding customizing font size option for docs.

I prefer devs to customise it with CSS, as it's more flexible this way.

You can also replace the footer component from `DocsPage`, if CSS solution not enough.
